site/content/en/references/kustomize/cmd/localize/_index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: "kustomize localize"
+linkTitle: "localize"
+type: docs
+weight: 9
+description: >
+  Downloads url content to a local directory, and replaces urls with paths to downloaded content
+---
+
+Disclaimer: This is an alpha command. Please see the [command proposal](https://github.com/kubernetes-sigs/kustomize/blob/master/proposals/22-04-localize-command.md)
+for full capabilities.
+
+### Description
+
+The `kustomize localize` command makes a recursive copy of a kustomization
```

Comment:
```suggestion
The `kustomize localize` command makes a recursive copy of a directory that contains a kustomization file,
```

Reply:
In general we should be very specific; `kustomization` can refer to many things, and most of the time refers to the Kustomization object itself. If you could please audit for anywhere else that we may be referring to `kustomization` here when what you mean is the directory itself, please update it to be very specific.

Reply:
I added the word `file` or `root` to any use of `kustomization` that doesn't refer to the `kustomization` object.

For this specific case, I reworded it to `kustomization root` and linked the definition of `kustomization root` since we've defined it on our site already and we were using this term to describe `target` later in the docs anyway: https://github.com/kubernetes-sigs/cli-experimental/pull/313/files#diff-3de085d06ca2f9e7f7dbd1c0c3c1f86786e2555ac850003aae4ab3875249b8d5R15